src/com/goide/runconfig/GoRunConfigurationBase.java

```diff
@@ -194,4 +195,14 @@ public String getWorkingDirectory() {
   public void setWorkingDirectory(@NotNull String workingDirectory) {
     myWorkingDirectory = workingDirectory;
   }
+
+  @NotNull
+  public String getFilePath() {
```

Comment:
The change to move filePath up to the base is probably unnecessary now.  I had originally used this to point to the app root directory until I looked up the source root and eliminated the file path from the run app config entirely.  Also, I totally overlooked the TestRun configs so I will fix these tomorrow.

Like I said, totally naive approach and I have never done Idea plugin work before so there are probably much better ways to do this later.